### ISSUE TITLE
Loss Function Feature

### ISSUE THREAD

**Issue by contepablod** [Qiskit/qiskit-machine-learning#500]
### What should we add?

Hi! Could you possibly explain in a markdown cell the connection between the quantum circuit, Loss Function and learning rate?

I'm trying to classify esophagus cancer images (Tensor shape [3,260,260] and I'm getting stucked in a 71% of accuracy.

If a NLLLoss is used, a SoftMax activation function is needed instead of a Relu?

Would a BCELoss function in a classifier perform better?

Thanks for reading!

Best.

**Comment by adekusar-drl** [comment]
The question is unclear without further details and explanations, honestly. Use of loss/activation functions depends on the design of the neural network. I guess you are talking about PyTorch networks? Do you have quantum layers?

Please also consider posting such questions to the Slack channels here: https://qiskit.slack.com

**Comment by adekusar-drl** [comment]
@contepablod have managed to solve the issue?

**Comment by contepablod** [comment]
Sure, I chose CrossEntropyLoss and I could deal with it. You can check my repo of that if you want.
Thanks!

**Comment by adekusar-drl** [comment]
Thank you. I'm closing the issue. Feel free to re-open if you still have the problem.
